package.json

```diff
@@ -78,6 +77,7 @@
     "ipfs-block-service": "~0.14.0",
     "ipfs-repo": "~0.22.1",
     "lodash": "^4.17.10",
+    "multihashes": "~0.4.13",
```

Comment:
I think `multihashes` is no needed after this very good change.

Reply:
It's [still being used in a test](https://github.com/ipld/js-ipld-dag-pb/blob/master/test/dag-node-test.js#L432) which is why it's moved from `dependencies` to `devDependencies`.